benchbuild/projects/benchbuild/linpack.py

```diff
@@ -1,27 +1,38 @@
 import logging
 
-from benchbuild import project
-from benchbuild.utils import compiler, download, path, run
+from plumbum import local
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unable to import 'plumbum'](https://app.codacy.com/manual/polyjit/benchbuild_2/pullRequest?prid=5028678)